readme.md

```diff
@@ -257,6 +257,7 @@
 - [Hyperapp](https://github.com/jorgebucaran/awesome-hyperapp#readme) - Tiny JavaScript library for building web apps.
 - [lit-html](https://github.com/web-padawan/awesome-lit-html#readme) - HTML templating library for JavaScript.
 - [JAMstack](https://github.com/automata/awesome-jamstack#readme) - Modern web development architecture based on client-side JavaScript, reusable APIs, and prebuilt markup.
+- [WordPress-Gatsby](https://github.com/henrikwirth/awesome-wordpress-gatsby#readme) - Web development technology stack with WordPress as a backend and Gatsby as a frontend.
```

Comment:
```suggestion
- [WordPress-Gatsby](https://github.com/henrikwirth/awesome-wordpress-gatsby#readme) - Web development technology stack with WordPress as a back end and Gatsby as a front end.
```

Reply:
Good catch

Reply:
@sindresorhus wouldn't it make sense to use front-end and back-end instead? Also in the Readme are two approaches `Frontend development` under `Platforms` and `Front-End Development` as a topic.